pkgs/by-name/cr/crusader/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  autoPatchelfHook,
+  lib,
+  libgcc,
+  libGL,
+  fetchFromGitHub,
+  fontconfig,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+  libxkbcommon,
+  xorg,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "crusader";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "Zoxc";
+    repo = "crusader";
+    tag = "v${version}";
+    hash = "sha256-M5zMOOYDS91p0EuDSlQ3K6eiVQpbX6953q+cXBMix2s=";
+  };
+
+  sourceRoot = "${src.name}/src";
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-f0TWiRX203/gNsa9UEr/1Bv+kUxLAK/Zlw+S693xZlE=";
+
+  # autoPatchelfHook required on linux for crusader-gui
+  nativeBuildInputs = [
+    pkg-config
+  ] ++ lib.optionals (!stdenv.isDarwin) [ autoPatchelfHook ];
+
+  buildInputs = [
+    fontconfig
+    libgcc
+    libxkbcommon
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXi
+  ];
+
+  # required for crusader-gui
+  runtimeDependencies = [
+    libGL
+    libxkbcommon
+  ];
+
```

Comment:
`passthru.updateScript = nix-update-script {}`

For auto updates

Reply:
Added.